datasets/opus_openoffice/README.md

```diff
@@ -0,0 +1,159 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- de
+- en_GB
+- es
+- fr
+- ja
+- ru
+- sv
+- zh_CN
```

Comment:
can you also format the languages depending on the config name if you don't mind ?

Sorry I didn't notice it in my previous review